docs/source/examples/simple-landscape-braket.md

```diff
@@ -0,0 +1,203 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Using ZNE to compute the energy landscape of a variational circuit with Braket
+
+This tutorial shows an example in which the energy landscape for a two-qubit variational circuit is explored with and without error mitigation, using Braket as our frontend.
+
+```{code-cell} ipython3
+import matplotlib.pyplot as plt
+import numpy as np
+
+from braket.devices import LocalSimulator
+from braket.circuits import Circuit, Noise, Observable
+
+from mitiq.zne import mitigate_executor
+from mitiq.zne.inference import RichardsonFactory
+```
+
+## Defining the ideal variational circuit in Qiskit
+
++++
+
+We define a function which returns a simple two-qubit variational circuit depending on a single parameter  $\gamma$ ("gamma").
+
+```{code-cell} ipython3
+def variational_circuit(gamma: float) -> Circuit:
+    """Returns a two-qubit circuit for a given variational parameter.
+
+    Args:
+        gamma: The variational parameter.
+
+    Returns:
+        The two-qubit circuit with a fixed gamma.
+    """
+    my_circuit = Circuit()
+    my_circuit.rx(0, gamma)
+    my_circuit.cnot(0, 1)
+    my_circuit.rx(1, gamma)
+    my_circuit.cnot(0, 1)
+    my_circuit.rx(0, gamma)
+    
+    return my_circuit
+```
+
+We can visualize the circuit for a particular $\gamma$ as follows.
+
+```{code-cell} ipython3
+circuit = variational_circuit(gamma=np.pi)
+print(circuit)
+```
+
+## Defining the executor functions with and without noise
+To use error mitigation methods in Mitiq, we define an executor function which computes the expectation value of a simple Hamiltonian $H=Z \otimes Z$, i.e., Pauli-$Z$ on each qubit. To compare to the noiseless result, we define both a noiseless and a noisy executor below.
+
+```{code-cell} ipython3
+# Observable to measure
+z = np.diag([1, -1])
+hamiltonian = np.kron(z, z)
+
+def noiseless_executor(circ: Circuit) -> float:
+    """Simulates the execution of a circuit without noise.
+
+    Args:
+        circ: The input circuit.
+
+    Returns:
+        The expectation value of the ZZ observable.
+    """
+    device = LocalSimulator('braket_dm')
+    # Evaluate the ZZ expectation value
+    circ.expectation(observable=Observable.Z() @ Observable.Z(), target=range(2))
+    
+    task = device.run(circ, shots = 0)
```

Comment:
Looks like `shots=0` is the default, so maybe we just leave it out unless you think it's worth keeping around for clarity. (I had to go look up what `shots=0` meant)
```suggestion
    task = device.run(circ)
```

Reply:
We can leave it out.